pyproject.toml

```diff
@@ -161,6 +160,8 @@ ignore = [
     "S106",   # flake8-bandit: hardcoded password
     "S603",   # flake8-bandit: subprocess-without-shell-equals-true
     "S320",   # flake8-bandit: xml parsing vulnerable to XML attacks
+    "UP007",  # forbid use of typing.Union and typing.Optional
+    "UP015",  # Unnecessary mode="r" argument in open()
```

Comment:
If you want, we can include these 2 as well. I just thought I didn't want to be too prescriptive, perhaps there are people who prefer e.g. `typing.Optional[X]` over `X | None`